### ISSUE TITLE
Move reply-to-post to a dedicated button

### ISSUE THREAD

**Issue by ShadowJonathan** [buresdv/Mlem#112]
Users seem to find the bottom bar annoying; https://beehaw.org/post/505992

**Comment by buresdv** [comment]
Reading the linked post, the user doesn't want a dedicated button for posting, but they want the bottom tab bar and top toolbar to be hidden upon scrolling

**Comment by ShadowJonathan** [comment]
Yes, but I also interpreted this feature request from that XY, but that's fair then
